docs/api/download-file.api.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: download-file
+title: "downloadFile"
+description: "downloadFile"
+sidebar_label: "downloadFile"
+hide_title: true
+hide_table_of_contents: true
+api: eJy9U0uP2yAQ/itoTq2E1kmP3PbQRqteqj5OUVQRexLTxZiFcdoI8d+rwY9mk90eeujJmBmG70WC3mPQZHr30ICCpv/pbK+bD8YiSPA66A4JQwS1TeB0h6AA3cmE3nXoCCQEfBpMwAYUhQElGAcKvKYWJMS6xU6DSkBnz0cjBeOOkLNcppXPv47Z8cHoexcxcv3dasWfBmMdjGdaoGCDJLSI5uiwEUOwgnoxExVaHIxFYZzQTlwwE/uzMBTFBK/uHTFflUB7b01dNKt+RL4h/QVhzllCxHoIhs5FRe3N90fk9Y7xkz6yusCSR9jJK/BXjnRIbc9OHZGKQdSCguq0ri6wV+niJ1dMMFaJmeRqnseyYjjN1g7BgoKWyEdVVantI2W4xgISTjoYvbej2tw1yn3QgyUewDu5+DJz/sLSjP0L80Um7c1HPMNkd4u6wQByjsbczxIad+jLQUOWaxvtzh02KO4/PdwAvSoyy7GwvlvdrSY7dc3gebbvI3W6GDndvMRjkv3Z+AU94S+qvNXGQZajhGmyZAunNcirp6Ke/xZfeHvK2GLNTk7SbiGlvY74LdiceftpwFCS88eJYl9jIq8bUAdtI94AXuILbz5PD+2tAPkykdkbdy6G2wFH74t1zyjxM/6PVxed8i7LOSlMfSzd1zV6ujh080o5k8vr2bz/ChL0wFZdhvGxhHFa8PQXAc25nHGVjMpXmlMaQ57z0j+WXj2xPIOxm5XZ5Zx/A4Xs7C0=
```

Comment:
Are these keys intended to be committed -- these seem auto-generated, so maybe expected.

Reply:
@mattcoganymede yeah they are autogenerated. some sort of uuid or hash it seems